texthero/representation.py

```diff
@@ -136,22 +56,18 @@ def count(
     min_df=1,
     max_df=1.0,
     binary=False,
-) -> pd.Series:
+) -> pd.DataFrame:
     """
     Represent a text-based Pandas Series using count.
 
-    Return a Document Representation Series with the
+    Return a Document Term DataFrame with the
```

Comment:
The newline formatting is a bit bizarre there + the docstring is not crystal clear.

What about something like that?

"""
Return a count document-term DataFrame based on the given Pandas Series

Rows of the returned DataFrame represent document whereas columns are terms ...


"""

Reply:
changed the docstring into a clearer version